doc/ale.txt

```diff
@@ -3482,6 +3482,7 @@ documented in additional help files.
   fortran.................................|ale-fortran-options|
     gcc...................................|ale-fortran-gcc|
     language_server.......................|ale-fortran-language-server|
+    fortran...............................|ale-fortran-fortitude|
```

Comment:
The tests will probably fail on the list of supported tools here because they are expected to be sorted alphabetically.